contracts/StablecoinConverter.sol

```diff
@@ -17,8 +17,8 @@ contract StablecoinConverter is EpochTokenLocker {
         bool isSellOrder,
         uint32 validFrom,
         uint32 validUntil,
-        uint256 buyAmount,
-        uint256 sellAmount
+        uint128 buyAmount,
```

Comment:
Are you sure we don't want `uint96` here (like we do in the other contracts)?

Reply:
yes, with uint128, we can store the order placement with 2 word (512 bit)stores in the EVM. reducing to uint96 would not change the picture.